uni/lib/view/Widgets/generic_expansion_card.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:flutter/material.dart';
+import 'package:expansion_tile_card/expansion_tile_card.dart';
+
+abstract class GenericExpansionCard extends StatefulWidget {
+  const GenericExpansionCard({Key? key}) : super(key: key);
+
+  @override
+  State<StatefulWidget> createState() {
+    return GenericExpansionCardState();
+  }
+
+  String getTitle();
+  Widget buildCardContent(BuildContext context);
+}
+
+class GenericExpansionCardState extends State<GenericExpansionCard> {
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+        margin: const EdgeInsets.symmetric(vertical: 10, horizontal: 20),
```

Comment:
Collapsed, it seems a bit too much vertical space to me. How about this?
```suggestion
        margin: const EdgeInsets.fromLTRB(20, 10, 20, 0),
```

Reply:
Great detail, I like it!